src/mongocxx/test/index_view.cpp

```diff
@@ -172,6 +172,52 @@ TEST_CASE("create_one", "[index_view]") {
         REQUIRE_NOTHROW(indexes.create_one(keys1.view(), options.view()));
         REQUIRE_THROWS_AS(indexes.create_one(keys2.view(), options.view()), operation_exception);
     }
+
+    SECTION("commitQuorum option") {
+        collection coll = db["index_view_create_one_commit_quorum"];
+        coll.drop();
+        coll.insert_one({});  // Ensure that the collection exists.
+        index_view indexes = coll.indexes();
+
+        auto key = make_document(kvp("a", 1));
+        index_model model(key.view());
+        options::index_view options;
+
+        auto commit_quorum_regex =
+            Catch::Matches("(.*)commit( )?quorum(.*)", Catch::CaseSensitive::No);
+
+        using namespace test_util;
+        bool is_supported = compare_versions(get_server_version(mongodb_client), "4.4") >= 0 &&
```

Comment:
The test failure on s390x seems related to the version of MongoDB running.

The logs of:
https://evergreen.mongodb.com/task/cxx_driver_ubuntu1604_zseries_compile_and_test_with_shared_libs_patch_510d2b0c04d1103bd1252088cda78522346b1f25_5f0f5f7f7742ae3af8c73ca8_20_07_15_19_56_48

say:
```
db version v4.3.0-2027-gbe045fe
```

That test is running against single, so `is_supported` would be false, and an exception is expected. But since that server version is likely reporting wire version 9, your wire version check is not getting hit. I think the right thing to do is change this comparison to check the wire version instead:

```
bool is_supported =  get_max_wire_version (mongodb_client) >= 9 && get_topology (mongoc_client) != "single";
```